geoportailv3/static/js/maincontroller.js

```diff
@@ -211,6 +219,30 @@ app.MainController = function(
       $('app-themeswitcher #themes-content').collapse('hide');
     }
   }, this));
+
+  var selectInteraction = new ol.interaction.Select({
+    filter: goog.bind(function(feature, layer) {
+      return this.drawnFeatures.getArray().indexOf(feature) != -1;
+    }, this)
+  });
+  this.map_.addInteraction(selectInteraction);
+
+  /**
+   * @type {ol.interaction.Select}
+   * @export
+   */
+  this.selectInteraction = selectInteraction;
+
+  // synchronize the drawn features collection with the feature overlay
+  var drawOverlay = ngeoFeatureOverlayMgr.getFeatureOverlay();
+  $scope.$watchCollection(goog.bind(function() {
+    return this.drawnFeatures.getArray();
+  }, this), goog.bind(function() {
+    drawOverlay.clear();
+    this.drawnFeatures.forEach(function(feature) {
+      drawOverlay.addFeature(feature);
+    });
+  }, this));
```

Comment:
As discussed I think the synchronization could/should be done by the feature overlay.